### PR TITLE
BCD indexer updates

### PR DESCRIPTION
**Pull request by harryttd**
Working BCD indexer on their 4.3.0 release. They have a 4.4.0 release but I don't want to default to that version yet bec BCD right now is primarily for DMV. DMV is running older protocol right now that won't work with latest BCD release.

### CODE REVIEW

File: charts/tezos/templates/_helpers.tpl [hunk 1]
```diff
@@ -112,11 +112,21 @@ metadata:
...
+  Has per-block votes defined?
+*/}}
+{{- define "tezos.hasPerBlockVotes" }}
+  {{- range .Values.protocols }}
+    {{- if .vote }}
+      {{- "true" }}
+    {{- end }}
+  {{- end }}
+{{- end }}
+
```
Comment (nicolasochem) [inline charts/tezos/templates/_helpers.tpl:125]: this looks like an unexpected change

**Review by nicolasochem — CHANGES_REQUESTED** [review]
As far as I can tell, this PR:

1. adds bcd pod to the tezos chart (optionally)
2. lints all python files it can find with black
3. re-adds a helm section that's gone from master (per_block_votes)

3. looks unvoluntary, should it be removed?

File: charts/tezos/templates/_helpers.tpl (hunk 1 above)
Comment (harryttd) [reply charts/tezos/templates/_helpers.tpl:125]: it's actually visually confusing the diff. this block is moved from the bottom of the file to here. Before the indexers

**Review by nicolasochem — APPROVED** [review]

**Comment by harryttd** [comment]
> As far as I can tell, this PR:
> 
> 1. adds bcd pod to the tezos chart (optionally)
> 2. lints all python files it can find with black
> 3. re-adds a helm section that's gone from master (per_block_votes)
> 4. looks unvoluntary, should it be removed?

yes to everything except 3 and 4. I responded on the inline comment

**Pull request merged**